src/core/crypto/multiEd25519.ts

```diff
@@ -184,6 +234,8 @@ export class MultiEd25519Signature extends Signature {
     return Hex.fromHexInput(this.toUint8Array()).toString();
   }
 
+  // endregion
```

Comment:
whats up with all those `region` `endregion`? is it something new?

Reply:
https://github.com/aptos-labs/aptos-ts-sdk/pull/147#discussion_r1486901932